app/views/sephora_style_guide/home/_pagination.slim

```diff
@@ -0,0 +1,128 @@
+article#pagination
+  h2.h2 Pagination Buttons
+
+  section
+    h3.h3 Top
+    br
+    label Less then 3 pages
+    br
+    nav.pagination
+      span.page.current
+        em 1
+      span.page
+        a 2
+      span.page.next
+        a
+          = render 'sephora_style_guide/shared/icon', icon_class: 'arrow', icon: 'right-arrow'
+    br
+    br
+    label Between pages
+    br
+    nav.pagination
+      span.page.prev
+        a
+          = render 'sephora_style_guide/shared/icon', icon_class: 'arrow', icon: 'left-arrow'
+      span.page
+        a 7
+      span.page.current
+        a 8
+      span.page
+        a 9
+      span.page.next
+        a
+          = render 'sephora_style_guide/shared/icon', icon_class: 'arrow', icon: 'right-arrow'
+    br
+    br
+    label On last page
+    br
+    nav.pagination
+      span.page.prev
+        a
+          = render 'sephora_style_guide/shared/icon', icon_class: 'arrow', icon: 'left-arrow'
+      span.page
+        a 206
+      span.page
+        a 207
+      span.page.current
+        a 208
+  section
+    h3.h3 Bottom
+    br
+    label Less then 3 pages - #F1F1F1
+    br
+    nav.pagination
+      span.page.current
+        em 1
+      span.page
+        a 2
+      span.page.next
+        a
+          = render 'sephora_style_guide/shared/icon', icon_class: 'arrow', icon: 'right-arrow'
+    br
+    br
+    
+    label On the first page
+    br
+    nav.pagination
+      span.page.prev
+        a
+          = render 'sephora_style_guide/shared/icon', icon_class: 'arrow', icon: 'left-arrow'
+      span.page.current
+        a 1
+      span.page
+        a 2
+      span.page
+        a ...
+      span.page
+        a 11
+      span.page
+        a 12
+      span.page.next
+        a
+          = render 'sephora_style_guide/shared/icon', icon_class: 'arrow', icon: 'right-arrow'
+    br
+    br
+    label Between pages
+    br
+    nav.pagination
+      span.page.prev
+        a
+          = render 'sephora_style_guide/shared/icon', icon_class: 'arrow', icon: 'left-arrow'
+      span.page
+        a 1
+      span.page
+        a ...
+      span.page.current
+        a 7
+      span.page
+        a 8
+      span.page
+        a 9
+      span.page
+        a ...
+      span.page
+        a 12
+      span.page.next
+        a
+          = render 'sephora_style_guide/shared/icon', icon_class: 'arrow', icon: 'right-arrow'
+    br
+    br
+    label On last page
+    br
+    nav.pagination
+      span.page.prev
+        a
+          = render 'sephora_style_guide/shared/icon', icon_class: 'arrow', icon: 'left-arrow'
```

Comment:
@saurabhsharma722 I think we are not on the same page on this. Instead of the concept of SVG sprites, we are going to have "inline SVGs". That is the server will render the required SVG directly without the need to "sprite" or explicitly call render or even edit the SVG file (beyond plain optimization). So how this line should look like is:
```ruby
= inline_svg 'sephora_style_guide/left.svg'
```